### ISSUE TITLE
Blockchain/buyer node report transaction completed, but seller node does not recognize payment

### ISSUE THREAD

**Issue by alahj** [kimitzu/kimitzu-client#81]
As "Complete Therapeutic Massage" ( PeerID Qmdgh8DfRQVNhhVvqE5zYQZU3ArgwpvefvHAFs4HRmbrVU) , I purchased a $10 in BTC for a "Sloth Massage" / "Test listings" from kimitzu_massage (PeerID: QmaaiLRgS9PxHV8re3Z2rhdKVzCxkSEQ9XVdpdK9rjkaiF).   The sale appeared on both Peers.  However, although the buyer ("Complete Therapeutic Massage") wallet says that the payment was sent (and Blockchain.com confirms that the transaction is complete), the payment does not appear in the seller's wallet.  (Note, there is an older existing transaction, from when I transferred BTC into the seller's wallet directly from electrum).  

![Screen Shot 2020-01-23 at 3 49 34 AM](https://user-images.githubusercontent.com/53591309/72982987-306b9600-3d95-11ea-8111-cc9ac5302e5a.png)
![Screen Shot 2020-01-23 at 3 56 42 AM](https://user-images.githubusercontent.com/53591309/72983006-38c3d100-3d95-11ea-87a3-f29d59e9a8b1.png)
![screencapture-blockchain-btc-tx-2aa188df2675850a80f9b2d9d99ccdf0c65767a639e377d27ca4038a2c1b020f-2020-01-23-03_57_43](https://user-images.githubusercontent.com/53591309/72983025-42e5cf80-3d95-11ea-8498-0f3732807e63.png)
![Screen Shot 2020-01-23 at 3 59 39 AM](https://user-images.githubusercontent.com/53591309/72983039-4b3e0a80-3d95-11ea-83ca-9d2aa36b4178.png)
![Screen Shot 2020-01-23 at 4 01 21 AM](https://user-images.githubusercontent.com/53591309/72983052-53964580-3d95-11ea-9d97-726636d13492.png)

**Comment by 24thsaint** [comment]
Hmm this looks like Openbazaar failed to hear the transaction.

Do you have access to the buyer node?

We'd need to send a POST request to `http://localhost:8100/wallet/resyncblockchain` to rescan the blockchain.

**Comment by alahj** [comment]
Yes, I have access to the buyer node.  Here's what I see on the buyer node when I go to the URL: 

![Screen Shot 2020-01-23 at 4 36 11 AM](https://user-images.githubusercontent.com/53591309/72985028-edf88800-3d99-11ea-8027-e2314ec9c52e.png)

**Comment by 24thsaint** [comment]
^Errata: This command should be ran on the seller node.

On the localhost page sir, do:

Right click -> Inspect -> Go to console.

Then the command:
`await fetch('http://localhost:8100/wallet/resyncblockchain', {method: 'POST'})`

For reference, you'll end up with something like:
![image](https://user-images.githubusercontent.com/8315544/72985537-255e4780-3e21-11ea-9f9e-7bad0e3158c3.png)

Then check the wallet page. :))

**Comment by alahj** [comment]
Trying...

**Comment by alahj** [comment]
Getting a Content Security Warning. 
![Screen Shot 2020-01-23 at 4 58 29 AM](https://user-images.githubusercontent.com/53591309/72986506-31082a80-3d9d-11ea-94e2-e3cb276ae318.png)

**Comment by 24thsaint** [comment]
Ohmy. Then we will need a REST client.

For example this one: https://install.advancedrestclient.com/install

Then, execute a POST request with URL:
`http://localhost:8100/wallet/resyncblockchain`

Image for reference:
![image](https://user-images.githubusercontent.com/8315544/72986894-2f357a00-3e24-11ea-8bec-0b83e2eb9e4a.png)

**Comment by alahj** [comment]
I tried a curl POST request: 

curl --request POST http://localhost:8100/wallet/resyncblockchain

Returned this:  {}

**Comment by alahj** [comment]
![Screen Shot 2020-01-23 at 5 24 16 AM](https://user-images.githubusercontent.com/53591309/72988182-a0cbe480-3da0-11ea-87fa-a27d6b712daf.png)

**Comment by alahj** [comment]
Wallet still does not display transaction.

**Comment by 24thsaint** [comment]
hmmm interesting... would you like to send over the `.kimitzu-ob/datastore` folder sir for further investigation?

**Comment by alahj** [comment]
Sure.

**Comment by 24thsaint** [comment]
We just need this folder, sir.

![Screenshot from 2020-01-23 21-52-38](https://user-images.githubusercontent.com/8315544/72990217-bdacfa00-3e2a-11ea-8403-ffd5f208b7df.png)

**Comment by 24thsaint** [comment]
Confirmed, transaction type is `offline order` which is still currently not supported as it has a different transaction flow as compared to an online order.

Reprioritizing to implement this in the next minor release.

---
## Dev Note
- Offline orders need to be confirmed. Indicators would be a `funded order` with `pending status`.
- Call /orderconfirmation endpoint, see [API](https://api.docs.openbazaar.org/?version=latest#f8f14b94-598e-f27d-6b8b-4c1b97d0ab23). Interface would show [Accept Order] and [Reject Order] and modify payload accordingly.
- Proceed with normal order flow.
